executor_manager/routers/routers.py

```
@@ -490,20 +490,87 @@ async def _handle_sandbox_callback(request: CallbackRequest):
+        return False
+
+    token = auth_header[7:]  # Remove "Bearer " prefix
+    secret_key = os.getenv("JWT_SECRET_KEY", "your-secret-key-here")
+    algorithm = os.getenv("JWT_ALGORITHM", "HS256")
+
+    try:
+        from jose import JWTError, jwt
+
+        # Just verify the token is valid, no need to check specific claims
+        jwt.decode(token, secret_key, algorithms=[algorithm])
+        return True
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)